mmdeploy/core/optimizers/function_marker.py

```diff
@@ -154,7 +154,7 @@ def impl(ys, prefix, level):
             if ys not in visit:
                 visit.add(ys)
                 root = ctx.names[ctx.index]
-                name = '.'.join(str(x) for x in (root, *prefix))
+                name = '/'.join(str(x) for x in (root, *prefix))
```

Comment:
I think it has been fixed in another PR. May rebase master

Reply:
Yes. Before there are a lot of conflicts. To make it clean, I reverted some commits. Maybe something wrong when revert the merge commit. `rebase` to master not work in this case.